examples/realworld/ext/auth/SignupPage.js

```diff
@@ -4,6 +4,8 @@ import { useHistory } from 'react-router-dom'
 import login from '@wasp/auth/login'
 import signup from '@wasp/auth/signup'
 
+import { errorMessage } from '@wasp/utils.js'
```

Comment:
This his not part of the Wasp JS API, right? That is a bit tricky!

Reply:
True! I removed that since it wasn't particularly good and just made a local helper version. Thanks 👍🏻